src/main/resources/db/migration/V29__iat_ddl.sql

```diff
@@ -0,0 +1,14 @@
+-- Creates the test package table
+
+CREATE TABLE IF NOT EXISTS test_package (
+  id                 UUID        NOT NULL,
+  package_id         TEXT        NOT NULL,
+  locked             BOOLEAN     NOT NULL,
```

Comment:
This should probably be `is_locked` to follow existing TIMS conventions.  I'll do a follow up PR to fix this.